program/src/state/stake.rs

```diff
@@ -56,15 +56,15 @@ impl Stake {
         self.discriminator.as_slice() == Stake::SPL_DISCRIMINATOR_SLICE
     }
 
-    pub fn new(authority: Pubkey, validator: Pubkey) -> Self {
+    pub fn new(authority: Pubkey, validator_vote: Pubkey) -> Self {
         Self {
             discriminator: Stake::SPL_DISCRIMINATOR.into(),
             amount: u64::default(),
             deactivation_timestamp: Option::default(),
             deactivating_amount: u64::default(),
             inactive_amount: u64::default(),
             authority,
-            validator,
+            validator: validator_vote,
```

Comment:
nit: can we have the field name be `validator_vote` for consistency and clarity?

Reply:
Done!